packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -126,8 +133,24 @@ of the iOS app that is using this module.''',
   @override
   String get name => 'release';
 
+  void _assertArgs() {
```

Comment:
If we're doing arg validation both here and in the Releaser subclasses, how should we choose where to validate the arg? I don't think this is wrong, I just think we need to make it clearer.

Reply:
Yeah, I spent some time wondering about that as well. I guess ideally validation of the args should stay in a single place.

But this one in particular is an arg that will be used by multiple releasers, so ended up adding this validation here so we don't repeat ourselves.

Wdyt? Does this makes sense? Maybe we can think in a better way to not repeat ourselves and to organize this validation?